cmd/dlv/main.go

```diff
@@ -28,6 +30,14 @@ var (
 )
 
 func main() {
+
+	// Config setup and load.
+	err := config.CreateConfigPath()
+	if err != nil {
+		fmt.Printf("Could not create config directory: %v.")
+	}
+	conf := config.LoadConfig()
```

Comment:
Maybe consolidate these steps (create / load) into just LoadConfig. That way, it can try and Stat the config dir or something, if it doesn't exist then create it, otherwise just load from it. Thoughts?